maskrcnn_benchmark/modeling/backbone/resnet.py

```diff
@@ -178,6 +199,7 @@ def _make_stage(
     num_groups,
     stride_in_1x1,
     first_stride,
+    dilation=1
```

Comment:
just double-checking if dilation is 1 here

Reply:
right, here dilation is 1

Reply:
Ok, I was just wondering if Detectron used 1 for dilation here in both cases, [and seems that they do use it](https://github.com/facebookresearch/Detectron/blob/7c0ad88fc0d33cf0f698a3554ee842262d27babf/detectron/modeling/ResNet.py#L293-L302), so it's good, thanks!